SUPLA/Cfg/AuthVC.swift

```diff
@@ -0,0 +1,312 @@
+/*
```

Comment:
Ultimately, I'd like to change Auth to AuthorizationSettings or something like that. The same goes for Android

Reply:
good moment to make such changes will be when implementing profiles, which will add some more screens on top of what is currently there.